tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -0,0 +1,100 @@
+import logging
+import re
+
+import pytest
+import random
+from tests.helpers import get_worker_nodes
+from ocs_ci.framework.pytest_customization.marks import tier4a
+from ocs_ci.ocs.resources import pod
+from ocs_ci.framework.testlib import (
+    tier4, ManageTest, aws_platform_required, ignore_leftovers, ipi_deployment_required
+)
+from ocs_ci.ocs import (
+    machine, constants, ocp, node
+)
+from tests.sanity_helpers import Sanity
+
+log = logging.getLogger(__name__)
+
+
+@tier4
+@tier4a
+@ignore_leftovers
+@aws_platform_required
+@ipi_deployment_required
+class TestNodeReplacement(ManageTest):
+    """
+    Knip-894 Node replacement - AWS-IPI-Proactive
+
+    """
+    @pytest.fixture(autouse=True)
+    def init_sanity(self):
+        """
+        Initialize Sanity instance
+
+        """
+        self.sanity_helpers = Sanity()
+
+    def test_nodereplacement_proactive(self, pvc_factory, pod_factory, dc_pod_factory):
```

Comment:
@PrasadDesala there is some duplicity with the test function from PR https://github.com/red-hat-storage/ocs-ci/pull/1491.

Please try avoiding this. An option would be to create a common function that will be called from both test methods

Reply:
@ebenahar 
@prsurve is working on this proactive pacth. 
I have submitted for Reactive PR #1607
So, you are suggesting us to cover both Proactive and reactive node replacement in a single test.py file?